src/components/core/Header.jsx

```diff
@@ -1,20 +1,29 @@
 import React from 'react';
 import './Header.scss';
 import githubIcon from '../../images/github-icon.svg';
+import githubIconWhite from '../../images/github-icon-white.svg';
+import Toggle from 'react-toggle';
+import { formatLabelForName } from '../../utils/formatting';
 
-const Header = () => (
+const Header = ({ onToggleChangeDarkMode, darkMode }) => (
   <div className="page-header">
     <h2 className="page-header-title">
       <span className="page-header-title-highlight">Issue</span>-Collab
     </h2>
-
+    <div>
+      <Toggle
+          onChange={onToggleChangeDarkMode}
+      /> 
+      <span className="toggle-label">{formatLabelForName("Dark Mode")}</span>
```

Comment:
Could we just have the text here instead of using a util?

Reply:
yes, i just thought that using your function will be better because you use it for the others Toggle components

Reply:
I think in those cases we need it because it is already lower-cased because it's coming from state, but in this case we can just type it as we want it 👍